src/forms/TextInput.jsx

```diff
@@ -33,6 +32,7 @@ export const TextInput = (props) => {
 		required,
 		requiredText,
 		validityMessage,
+		refCallback,
```

Comment:
we haven't added flow typing to this repo yet, so I'd recommend adding a proptype for this prop